### PR TITLE
My contributuion

### PR DESCRIPTION
**Pull request by aigerimdev**
Hi Aikol can you check my code?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
     <title>StartApp</title>
-</head>
-<body>
+  </head>
+  <body>
     <h1>Million Dollar Page!</h1>
-</body>
-</html>
\ No newline at end of file
+    <h2>New changs from Aigerim</h2>
```
Comment (aikoldolubaev) [inline index.html]: This text looks wrong, please fix your mistake!

Comment (aigerimdev) [reply index.html]: Ok I will try

Comment (aigerimdev) [reply index.html]: I fixed it

**Review by aikoldolubaev — APPROVED** [review]
Aigerim, great job!

**Pull request merged**